Welthungerhilfe/cgm-ml
# Rgbd multi

# Description
Previous PR134 changes

##How to run/test it?
1.) Make a input directory as a list of qrcodes. Make sure the qrcodes dir tree as mentioned in readme.
2.) Refer to the readme for the command to be used to run rgbd.py to create RGBD images in the output dir that you prefer.

User story # (US link)
https://dev.azure.com/cgmorg/ChildGrowthMonitor/_workitems/edit/175

# Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ x] My Code is review by 1 People
